Gemfile

```diff
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
-version_file = File.expand_path("../.rails-version", __FILE__)
+version_file = File.expand_path('.rails-version', __dir__)
 RAILS_VERSION = ENV['RAILS_VERSION'] || (File.exist?(version_file) && File.read(version_file).chomp) || ""
 
 gemspec
 
-rspec_dependencies_gemfile = File.expand_path("../Gemfile-rspec-dependencies", __FILE__)
+rspec_dependencies_gemfile = File.expand_path('Gemfile-rspec-dependencies', __dir__)
 eval_gemfile rspec_dependencies_gemfile
```

Comment:
Minor: does it need an absolute path?

Reply:
Not sure, followed recommendation from [here](https://github.com/rubocop-hq/rubocop/blob/d72591b7ddf3bb10d5805b4bd2c3d27d9d2ebd49/config/default.yml#L2693)

Reply:
It's cool and completely optional. Just wondering if plain `eval_gemfile 'Gemfile-rspec-dependencies'` would work.